Backpack/src/main/java/net/skyscanner/backpack/util/extensions.kt

```diff
@@ -0,0 +1,17 @@
+package net.skyscanner.backpack.util
+
+import android.animation.AnimatorInflater
+import android.os.Build
+import android.view.View
+import androidx.annotation.DrawableRes
+
+fun View.isStateListAnimatorSupported() =
+  if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.LOLLIPOP) {
+    false
+  } else isSpecificDeviceSupported()
+
+private fun isSpecificDeviceSupported() = Build.MODEL != "GT-I9505"
+
+fun View.loadStateListAnimator(@DrawableRes animator: Int) {
```

Comment:
Also internal. Plus it's worth consideration to move all these methods to BpkButton

Reply:
yup